homeworks/sasha.demidenko_SashaDemi/GamesProject/games.html

```diff
@@ -95,6 +95,10 @@ <h2 class="title-name" data-type="title">Overwatch 2</h2>
                     </article>
                 </template>
             </div>
+            <div id="loading-spinner" style="display: flex;justify-content: center;font-size: 35px;">
```

Comment:
Hey, @SashaDemi, inline styles are used in case, when you can't use normal styling.
Why don't you like to add these styles in your project css files?